### DISCUSSION TITLE
Generating images one frame at a time?

### DISCUSSION THREAD

**Discussion by aaronsantiago** [volotat/SD-CN-Animation#60] [Ideas]
Title. For a few use cases it would be useful to have finer grained control over what goes in without having to encode the inputs as video

**Comment by volotat** [comment]
Interesting proposal. I will consider making an ui for this, but don't expect to see it too soon.

**Reply by aaronsantiago** [reply]
Thoughts? A few use cases I have:

- Generating animations for games
- Applying this approach for consistency in live SD feeds
- Granularly changing interpolation framerate on image sequences

Personally I wouldn't use this via the UI and would mainly use it via the API, which I would assume most people who use this sort of feature would do as well. That may make implementation easier.
